plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -294,9 +323,9 @@ func assertRecord(t *testing.T, actualRecord arrow.Record, expected []testCase)
 	if len(expected) != int(actualRecord.NumCols()) {
 		t.Fatalf("expected record to have %d columns, got %d", len(expected), actualRecord.NumCols())
 	}
-	schema := actualRecord.Schema()
+	sc := actualRecord.Schema()
```

Comment:
Need to prevent shadowing the new import `"github.com/cloudquery/plugin-sdk/v4/schema"`